cores/cv32e40p/user_manual/source/intro.rst

```diff
@@ -110,8 +110,18 @@ CV32E40P currently supports the following features according to the RISC-V Privi
 * Hardware Performance Counters as described in :ref:`performance-counters` based on ``NUM_MHPMCOUNTERS`` parameter
 * Trap handling supporting direct mode or vectored mode as described at :ref:`exceptions-interrupts`
 
+
+Synthesis guidelines
+--------------------
+
+The CV32E40P core has been taped-out in several nodes (from 130nm to 22nm)
```

Comment:
This remark is true for RI5CY. It is probably not true for CV32E40P; I would leave it out all together as it belongs in marketing material as opposed to the user manual.